dropbox-cleaner.sh

```diff
@@ -1,63 +1,44 @@
 #!/bin/bash
+'''References:
+http://stackoverflow.com/questions/20723868/batch-rename-dropbox-conflict-files
+https://community.spiceworks.com/how_to/111611-how-to-change-all-files-folders-to-lowercase-unix
+'''
 
-#Thx, http://stackoverflow.com/questions/20723868/batch-rename-dropbox-conflict-files
```

Comment:
I prefer to preserve original post link, anyway all your other reference are very informative! :)

Reply:
Sure! let's preserve the original post link then.